Max-Lewis-table.py

```diff
@@ -0,0 +1,74 @@
+import sqlite3
+
+# Connect to the SQLite database
+conn = sqlite3.connect('/Users/haatlason/Documents/GitHub/sqlite-greyjoy/capstone-ironislands/f1db.db')
```

Comment:
Breyta í relative path
```suggestion
conn = sqlite3.connect('f1db.db')
```